.github/workflows/run_tests.yml

```diff
@@ -146,52 +168,23 @@ jobs:
         run: python -m pip install -e .[dev,xarray]
 
       - name: Build and install test dependencies
-        run: make lcov && make catch2
-
-      - name: Install docker, redis-server, and RedisAI
         run: |
-          sudo apt-get -y update && sudo apt-get -y install curl gnupg lsb-release software-properties-common ca-certificates && \
-          # Add latest redis to apt sources
-          sudo echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list && \
-          sudo curl -fsSL https://packages.redis.io/gpg | gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg && \
-          # Add latest docker to apt sources
-          sudo mkdir -m 0755 -p /etc/apt/keyrings &&
-          sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg &&
-          echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
-                $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null &&
-          # Install and update
-          sudo apt-get -y update && \
-          sudo apt-get -y install iputils-ping docker-ce docker-ce-cli containerd.io redis-tools=6:6.2.5-1rl1~focal1 redis-server=6:6.2.5-1rl1~focal1 &&
-          CONTAINER_NAME="redisai_$RANDOM" &&
-          sudo docker create --name $CONTAINER_NAME --rm redislabs/redisai:${{ matrix.rai_v }}-cpu-bionic && \
-          sudo mkdir -p /usr/lib/redis/modules/ &&
-          sudo docker cp $CONTAINER_NAME:/usr/lib/redis/modules/redisai.so /usr/lib/redis/modules &&
-          sudo docker cp $CONTAINER_NAME:/usr/lib/redis/modules/backends/ /usr/lib/redis/modules/ &&
+          make test-deps &&
```

Comment:
I didn't do this before because it is slower than grabbing from the docker container. As it is, explicitly calling make on test-deps is redundant since it will occur as needed with any `make test*` step

Reply:
The container that we were extracting RedisAI from uses Redis 6, whereas SmartSim is currently using Redis 7. Thus, these modules are now inconsistent with the redis-server that we build. While we could potentially continue testing with Redis 6, upgrading to Redis 7 ensures consistency with SmartSim